txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) (verifyAddress bool) {
+	_, err := dcrutil.DecodeAddress(address, wallet.chainParams)
+	if err != nil {
+		return
+	}
```

Comment:
If an error occurs, the function must return.

Reply:
While you don't **need** to return a value when using named return variables (and I agree that using a named return variable makes the `tx.AddSendDestination` method easier to understand), returning an explicit value in this instance makes more readable sense.

```suggestion
func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) (verifyAddress bool) {
	_, err := dcrutil.DecodeAddress(address, wallet.chainParams)
	if err != nil {
		return false
	}
```